index.html

```diff
@@ -1,268 +1,290 @@
-                        <div class="arrow">
-                            <p class="skill"> Languages</p>
-                            <p> <i class="fa fa-chevron-down" aria-hidden="true"></i> </p>
-                        </div> 
-
-                        <ul class ="languages">
-                            <li><img src="Images/JavaScript.jpg" alt=""><a href="#" class = "tech">JavaScript </a></a></li>
-                            <li> <img src="Images/html.jpg" alt=""><a href="#" class = "tech" >HTML5</a></li>
-                            <li><img src="Images/css.jpg" alt=""><a href="#" class = "tech">CSS</a></li>
-                        </ul>
+                        <div>
+                            <ul class="exp">
+                                <li> Canopy &nbsp;</li>
+                                <li class="exp-text"><img src="Images/counter.png" alt=""> &nbsp;Back End Dev&nbsp;</li>
+                                <li class="exp-text"><img src="Images/counter.png" alt=""> &nbsp;2015</li>
+                            </ul>
+                        </div>
 
+                        <div>
+                            <p class="job-info">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
                         </div>
-                    </li>
 
-                    <li>
-                        <div class="arrow">
-                            <p class="skill">Frameworks</p>
-                            <p> <i class="fa fa-chevron-right" aria-hidden="true"></i></p>                            
+                        <div>
+                            <ul class="skill">
+                                <li><button>HTML</button></li>
+                                <li><button>CSS</button></li>
+                                <li><button>JavaScript</button></li>
+                            </ul>
                         </div>
-                        <hr id="line-1">
 
-                    </li>
+                        <div class="project-button">
+                            <button>See project</button>
+                        </div>
+                    </div>
 
-                    <li>
-                        <div class="arrow"> 
-                            <p class="skill">Skills</p> <p> <i class="fa fa-chevron-right" aria-hidden="true"></i> </p>
+                    <div class="card-img">
+                        <img src="Images/Snapshoot Portfolio_4.svg" alt="Test">
+                    </div>
+                </div>    
+            </section> 
+
+            <section class="about">
+                <div class="abt-desk">
+                    <div class="sct-1">
+                        <h2>About Myself</h2>                
+                        <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
```

Comment:
Kindly fix the font-size of this text. We need to follow the Figma design guideline for each CSS values 👍 
<img width="279" alt="Screenshot 2023-04-07 at 10 33 18 PM" src="https://user-images.githubusercontent.com/25887253/230671150-b17a5057-16d9-477f-809b-8b39016240c9.png">
<img width="784" alt="Screenshot 2023-04-07 at 10 34 20 PM" src="https://user-images.githubusercontent.com/25887253/230671162-eea50099-1d2e-443d-b01d-f64833b59000.png">